### PR TITLE
Feat/melih branch mvp api

### PR DESCRIPTION
**Pull request by Sirmelihy**
Hi,

I made some changes to our API. It now provides the results information as we wanted. I believe the API is currently sufficient for our beta version. We will continue to make improvements and changes as we receive feedback.

Please review the commits carefully.

### CODE REVIEW

File: Rater.Domain/DataTransferObjects/MetricDto/MetricOfParticipantWinnerDto.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rater.Domain.DataTransferObjects.MetricDto
+{
+    public class MetricOfParticipantWinnerDto
```
Comment (fehmisener) [inline Rater.Domain/DataTransferObjects/MetricDto/MetricOfParticipantWinnerDto.cs]: This is not a good class name it shouldn't contain words like `of`

Comment (fehmisener) [reply Rater.Domain/DataTransferObjects/MetricDto/MetricOfParticipantWinnerDto.cs]: And you don't need to make everything class

Comment (fehmisener) [reply Rater.Domain/DataTransferObjects/MetricDto/MetricOfParticipantWinnerDto.cs]: We're trying to work in Object Oriented methodology not lexical/verbal talking

File: Rater.Domain/DataTransferObjects/MetricDto/MetricWinnerDto.cs [hunk 2]
```diff
@@ -7,14 +7,14 @@
 
 namespace Rater.Domain.DataTransferObjects.MetricDto
 {
-    public class MetricResponseForResultDto
+    public class MetricWinnerDto
```
Comment (fehmisener) [inline Rater.Domain/DataTransferObjects/MetricDto/MetricWinnerDto.cs]: TBH, I really didn't like this `Winner` keyword. It's not a competition it's should be something like best or etc.

**Review by fehmisener — CHANGES_REQUESTED** [review]
One last question: Do we have token authentication for this result services? User must access space with the password then be able to call these services with the token that retrieved by the token service

File: Rater.Domain/DataTransferObjects/MetricDto/MetricOfParticipantWinnerDto.cs (hunk 1 above)
Comment (Sirmelihy) [reply Rater.Domain/DataTransferObjects/MetricDto/MetricOfParticipantWinnerDto.cs]: Yes Resulting needs token you can check SpaceController

**Review by fehmisener — APPROVED** [review]

**Pull request merged**